.github/workflows/ci_performance_metrics_monitoring.yml

```diff
@@ -74,6 +75,9 @@ jobs:
       - run: bundle exec rails assets:precompile
         name: Precompile assets
         working-directory: ./development_app/
+      - run: rm ./config/webpacker.yml; cp ./tmp/webpacker_runtime.yml ./config/webpacker.yml
```

Comment:
this is a Temporary fix until https://github.com/shakacode/shakapacker/pull/254 is getting merged and released.